doc/arkode/guide/source/Usage/LSRKStep/User_callable.rst

```diff
@@ -239,12 +286,28 @@ Allowable Method Families
 
    **Return value:**
       * *ARK_SUCCESS* if successful
-      * *ARKLS_MEM_NULL* if ``arkode_mem`` was ``NULL``.
+      * *ARK_MEM_NULL* if ``arkode_mem`` was ``NULL``.
 
 .. note:: If LSRKStepSetDomEigSafetyFactor routine is not called, then the default ``dom_eig_safety`` is
    set to :math:`1.01`. Calling this function with ``dom_eig_safety < 1`` resets the default value.
 
 
+.. c:function:: int LSRKStepSetNumSucceedingWarmups(void* arkode_mem, int num_succ_warmups);
+
+   Specifies the number of the preprocessing warmups before each estimate call succeeding the very first estimate call.
+
+   **Arguments:**
+      * *arkode_mem* -- pointer to the LSRKStep memory block.
+      * *num_succ_warmups* -- the number of succeeding warmups.
+
+   **Return value:**
+      * *ARK_SUCCESS* if successful
+      * *ARK_MEM_NULL* if ``arkode_mem`` was ``NULL``.
+
+.. note:: If LSRKStepSetNumSucceedingWarmups routine is not called, then the default ``num_succ_warmups`` is set to :math:`0`.
+   Calling this function with ``num_succ_warmups < 0`` resets the default.
```

Comment:
I'd indent this so that the "Note" is included as part of this function's documentation:
```suggestion
   .. note:: If LSRKStepSetNumSucceedingWarmups routine is not called, then the default ``num_succ_warmups`` is set to :math:`0`.
      Calling this function with ``num_succ_warmups < 0`` resets the default.
```